adal/src/telemetry/java/com/microsoft/aad/adal/BrokerEvent.java

```diff
@@ -57,6 +57,32 @@ void setBrokerAccountServiceConnected() {
         setProperty(EventStrings.BROKER_ACCOUNT_SERVICE_CONNECTED, Boolean.toString(true));
     }
 
+    void setServerErrorCode(final String errorCode) {
+        if (null != errorCode && !errorCode.equals("0")) {
+            setProperty(EventStrings.SERVER_ERROR_CODE, errorCode.trim());
+        }
+    }
+
+    void setServerSubErrorCode(final String subErrorCode) {
+        if (null != subErrorCode && !subErrorCode.equals("0")) {
```

Comment:
Same above.

Reply:
👍

Reply:
Fixed in d2dd52acf54853cb395b40375135d8aebcc28741